src/components/links.svelte

```diff
@@ -1,9 +1,3 @@
-<script lang="ts">
```

Comment:
This was something I had forgotten to take out from a svelte doc example in a previous PR. It doesn't do anything for us.